src/vs/workbench/browser/parts/editor/media/tabstitle.css

```diff
@@ -41,7 +41,6 @@
 	cursor: pointer;
 	height: 35px;
 	box-sizing: border-box;
-	border: 1px solid transparent;
```

Comment:
@shizengzhou not a good change, this border is needed when the `tab.border` color is set, see:

![image](https://user-images.githubusercontent.com/900690/39416426-1942de88-4c4d-11e8-90c0-a1e1ea624fd2.png)